package.json

```diff
@@ -65,7 +65,7 @@
     "clean-webpack-plugin": "^0.1.19",
     "codecov": "^3.0.0",
     "css-loader": "^0.28.9",
-    "enzyme": "^3.3.0",
+    "enzyme": "^3.6.0",
```

Comment:
Note to Reviewers: This is needed to test the ErrorBoundary. Please see https://github.com/airbnb/enzyme/issues/1255